udf/pos_of_trailing_set_bit.sql

```diff
@@ -10,12 +10,12 @@ see https://stackoverflow.com/a/42747608/1260237
 */
 
 CREATE TEMP FUNCTION
-  udf_bitpos( bits INT64 ) AS ( CAST(SAFE.LOG(bits & -bits, 2) AS INT64));
+  udf_pos_of_trailing_set_bit( bits INT64 ) AS ( CAST(SAFE.LOG(bits & -bits, 2) AS INT64));
```

Comment:
For naming, I think I chose `bitpos` originally in analogy to BigQuery's STRPOS function, although the direction and the indexing are different.

`pos_of_trailing_set_bit` is a comprehensive name, but I would like it to be shorter if possible. What do you think of `bitpos_rightmost` or `bitpos_trailing`? Or does `bitpos` seem to obtuse in the first place?

Reply:
Analogy to `STRPOS` I understand, but imho `bitpos` is confusing to see, since it doesn't clarify that it's finding set bits.

I _believe_ this will be a largely internally-used function, so I'm okay with a larger, comprehensive name. If you see a need for it outside of these few views/UDFs then we should shorten it.